textacy/viz/termite.py

```diff
@@ -200,3 +209,131 @@ def draw_termite_plot(
+        n_terms=25,
+        rank_terms_by="max",
+        sort_terms_by="seriation",
+        save=False,
+        rc_params=None,
+    ):
+        """
+        Make a "termite" plot for assessing topic models using a tabular layout
+        to promote comparison of terms both within and across topics.
+        Args:
+            components (:class:`pandas.DataFrame` or sparse matrix): corpus
+                represented as a term-topic matrix with shape (n_terms, n_topics);
+                should have terms as index and topics as column names
+            topics (int or Sequence[int]): topic(s) to include in termite plot;
+                if -1, all topics are included
+            highlight_topics (str or Sequence[str]): names for up to 6 topics
+                to visually highlight in the plot with contrasting colors
+            n_terms (int): number of top terms to include in termite plot
+            rank_terms_by ({'max', 'mean', 'var'}): argument to dataframe `agg`
+                function, used to rank terms;
+                the top-ranked ``n_terms`` are included in the plot
+            sort_terms_by ({'seriation', 'weight', 'index', 'alphabetical'}):
+                method used to vertically sort the selected top ``n_terms`` terms;
+                the default ("seriation") groups similar terms together, which
+                facilitates cross-topic assessment
+            save (str): give the full /path/to/fname on disk to save figure
+                rc_params (dict, optional): allow passing parameters to rc_context
+                in matplotlib.plyplot, details in
+                https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.pyplot.rc_context.html
+        Returns:
+            ``matplotlib.axes.Axes.axis``: Axis on which termite plot is plotted.
+        Raises:
+            ValueError: if more than 6 topics are selected for highlighting, or
+                an invalid value is passed for the sort_topics_by, rank_terms_by,
+                and/or sort_terms_by params
+        References:
+            - Chuang, Jason, Christopher D. Manning, and Jeffrey Heer. "Termite:
+              Visualization techniques for assessing textual topic models."
+              Proceedings of the International Working Conference on Advanced
+              Visual Interfaces. ACM, 2012.
+            - Fajwel Fogel, Alexandre d’Aspremont, and Milan Vojnovic. 2016.
+              Spectral ranking using seriation. J. Mach. Learn. Res. 17, 1
+              (January 2016), 3013–3057.
+        See Also:
+            :func:`viz.termite_plot <textacy.viz.termite.termite_plot>`
+        TODO: `rank_terms_by` other metrics, e.g. topic salience or relevance
+        """
+        if highlight_topics is not None:
+            if isinstance(highlight_topics, str):
+                highlight_topics = [highlight_topics]
+            elif len(highlight_topics) > 6:
+                raise ValueError("no more than 6 topics may be highlighted at once")
```

Comment:
is this a strict technical requirement, or just an aesthetic preference?

Reply:
this is actually a hold-over from your function (see line 138). 

Its presumably because you hard-code the available colors, so additional complementary colors for highlighting is unavailable. 

I honestly would avoid this alltogether and adopt a `seaborn`-esque approach: the function accepts a list of highlighted features/columns (+rows?), and a colormap/iterator that gets applied to them. Let the user worry about which those are, as long as we apply a decent default.